d3b_utils/aws_bucket_contents.py

```diff
@@ -0,0 +1,81 @@
+import boto3
+import pandas as pd
+
+
+def fetch_aws_bucket_obj_info(bucket_name, path_to_files):
+    """
+    List the objects of a bucket and the size of each object. Returns a
+    dict. Inspiration from
```

Comment:
Not a dict. Also I think generally we use a :return: and :rtype: docstring elements